pkg/bgp/speaker/log.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package speaker
```

Comment:
What's the point of this new file? Can't the logger be used directly?

Reply:
The point of the file is to keep all the irrelevant logger code away from the main code. @twpayne I've moved the logging wrapper of MetalLB to a shared package